README.md

```diff
@@ -1,76 +1,80 @@
-[5]: https://cloud.google.com/code/docs/vscode
-[6]: https://github.com/GoogleCloudPlatform/cloud-code-samples
-[7]: https://github.com/GoogleCloudPlatform/cloud-code-vscode/issues/new?assignees=&labels=&template=bug_report.md&title=
-[8]: https://github.com/GoogleCloudPlatform/cloud-code-vscode/issues/new?assignees=&labels=enhancement&template=feature_request.md&title=
-[9]: https://cloud.google.com/code
-[10]: https://cloud.google.com/code/docs/vscode/install
-[11]: https://cloud.google.com/code/docs/vscode/troubleshooting
-[12]: https://github.com/GoogleCloudPlatform/cloud-code-vscode/raw/master/images/app_deploy_debug.gif
-[13]: https://join.slack.com/t/googlecloud-community/shared_invite/zt-erdf4ity-8ZMUQ18DYV~5hkbZ~gCswg
-[14]: https://www.gstatic.com/cloudssh/cloudcode/client-library-browser.gif
-[15]: https://www.gstatic.com/cloudssh/cloudcode/secret-manager-explorer.png
+Cloud Code for VS Code brings the power and convenience of IDEs to cloud-native Kubernetes and Cloud Run application development. Cloud Code works with Google’s command-line [container tools](https://github.com/GoogleContainerTools) like [`skaffold`](https://skaffold.dev/) and [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/) under the hood, providing local, continuous feedback on your project as you build, edit, and run your applications locally or in the cloud.
+
+![E2E workflow](https://github.com/GoogleCloudPlatform/cloud-code-vscode/raw/master/images/cloud-code-quick-deploy.gif)
+
+List of Supported Services
+- Kubernetes & Google Kubernetes Engine
+- Cloud Run 
+- Secret Manager
+- Cloud APIs 
+- Cloud Logging 
+- Cloud Build 
+- Kubernetes Config Connector
+- Migrate for Anthos
+
+# Key Features
+
+## Supporting your development workflow
+
+Get run-ready samples, out-of-the-box configuration snippets, support for key Google services like Google Cloud APIs and Cloud Build, one click deployment, a tailored debugging experience, and much more with Cloud Code — making developing with Kubernetes and Cloud Run a whole lot easier!
+
+<details>
+  <summary>Read more</summary>
+  
+  ### Highlights:
+  - Pick your preferred language with Cloud Code’s support for Go, Java, Node.js, Python, .NET Core app development.
+</details>
+
+## Kubernetes development
+
+Create and run a new app in minutes with Cloud Code’s Kubernetes support. Or work on an existing application, customize its YAML with Cloud Code’s intelligent authoring support, debug it relentlessly with the setup-free debugger, and run it on any of your Kubernetes clusters. Whatever your workflow is, Cloud Code helps you spend less time on configuration and context-switching, so you can focus on developing your app.
+
+<details>
+  <summary>Read more</summary>
+
+  ### Highlights:
+  - Get started with in-built ready-to-run starter Kubernetes apps for your favorite languages and frameworks. [Read the docs](https://cloud.google.com/code/docs/vscode/creating-an-application)
```

Comment:
I've been told in-built is a ye olde English word. 'built-in' here.

Reply:
Done